### PR TITLE
Avoid double slashes in attachemnts urls

### PR DESCRIPTION
**Pull request by tmotyl**
With this patch attachments url will not have doubled slash in URL:
/wiki//download/attachments/

after patch:
/wiki/download/attachments/

### CODE REVIEW

**Comment by gergelykalman** [comment]
1) I fail to see what the problem is, can you be more specific?
2) The fix itself is a very bad one, everything has to go through pathlib or os.path.* at the very least to be portable.

**Comment by tmotyl** [comment]
@gergelykalman let me rephrase the description:
Right now your script generates a path with double slash:
/wiki**//**download/attachments/

While it should be just one slash.
Feel free to improve the patch as you wish. I don't see why os.path should be used as we're talking about URL and not path on the disk.

**Comment by gergelykalman** [comment]
Yeah, you just copy-pasted the exact same lines without clarifying anything. Where is this url that is being generated? I assumed it was the url that got turned into a path, which would make this an error worth handling. While double slashes in an actual http url are not great it's not really important.
